Common/Brokerages/BinanceBrokerageModel.cs

```diff
@@ -121,6 +121,13 @@ public override bool CanSubmitOrder(Security security, Order order, out Brokerag
             {
                 case LimitOrder limitOrder:
                     quantityIsValid &= IsOrderSizeLargeEnough(limitOrder.LimitPrice);
```

Comment:
You could save `price` here: `price = limitOrder.LimitPrice`, and in every `case` accordingly. It would avoid duplication.